cors-policy/pom.xml

```diff
@@ -24,9 +24,28 @@
       <artifactId>apiman-gateway-engine-policies</artifactId>
       <scope>provided</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
```

Comment:
Yeah let's make it a proper parent and manage all our dependency versions there instead of in each plugin.